apis/flowcollector/v1beta2/flowcollector_types.go

```diff
@@ -84,6 +84,21 @@ type FlowCollectorSpec struct {
 	// +optional
 	// +k8s:conversion-gen=false
 	Exporters []*FlowCollectorExporter `json:"exporters"`
+
+	// `networkPolicy` define network policy settings for netobserv
+	// +k8s:conversion-gen=false
+	NetworkPolicy NetworkPolycy `json:"networkPolicy,omitempty"`
+}
+
+type NetworkPolycy struct {
+	// Set `deploy` to `false` to disable network policy deployment. It is enabled by default.
```

Comment:
We could add more information about what this policy does, such as:
> This network policy better isolates the NetObserv components to prevent undesired connections to them. It is recommended to install it.

(or something in that vein)

Also, shouldn't we name that `Enable`, like we do generally for anything that can be turned on/off ?